frontend/packages/integration-tests-cypress/tests/crud/namespace-crud.cy.ts

```diff
@@ -77,7 +77,6 @@ describe('Namespace', () => {
     projectDropdown.shouldContain(defaultProjectName);
     detailsPage.breadcrumb(0).contains('Services').click();
     listPage.rows.shouldBeLoaded();
-    projectDropdown.shouldContain(allProjectsDropdownLabel);
```

Comment:
When we click on a list item with a namespace set in the URL, the project dropdown will automatically select that ns, as it continuously monitors the ns in the URL. 
So, I've removed this line from the test.